tasks/internal/update-fbc-catalog-task/update-fbc-catalog-task.yaml

```diff
@@ -134,6 +152,19 @@ spec:
           echo "${build}"
         }
 
+        create_auth_file() {
+          mkdir -p "${HOME}/.config/containers"
+          targetIndex="$(params.targetIndex)"
+          authName="${targetIndex%:*}"
+
+          # disabling debug to not leak the token
+          set +x
+          jq --arg authName "${authName}" \
+             --arg token "$(base64 -w 0 < <(printf %s "${TARGET_INDEX_CREDENTIAL}"))" \
```

Comment:
What's the `printf` for? Why not just `<<< "${TARGET_INDEX_CREDENTIAL}"`? Also, what's the `-w 0` for? According to docs, it's "Wrap encoded output after column.". I tried it with and without and didn't see any difference.

Reply:
the `<<<` adds a "\n" in the result causing the password to not match due to the extra newline. I trying was trying to find out a way avoid the password to leak even when using `set -x`. This was the last try - which ended up not working as expected as the other tests (with echo -n for instance). I accept suggestions for this one.

The -w 0 is a workaround to make the base64 string in a single line. By default it was wrapping in two, at least. I did not want to use tr or sed, or anything else to unwrapp it.

Reply:
try this:

`echo "myrobot+myuser:mylongpasswordtobebase64encodedanditshouldbeverylongtomakedifference" | base64` and afterwards with the `base64 -w 0`

Reply:
Thanks for the explanation. Interesting with the new line - you're right, the result is different!

And the wrapping - it does not happen for me. But I guess it can depend on the version of base64. So ok to have it.